### PR TITLE
update command line options documentation

### PR DESCRIPTION
**Pull request by phemmer**
This section of the doc was fairly out of date.
Updated it, as well as added a test to ensure any future changes help changes are properly documented.

### CODE REVIEW

File: kube-router.go [hunk 1]
```diff
@@ -10,10 +10,20 @@ import (
...
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/pflag"
 )
 
 func main() {
+	if err := Main(); err != nil {
+		fmt.Fprintf(os.Stderr, "%s\n", err)
+		os.Exit(1)
+	}
+	os.Exit(0)
```
Comment (phemmer) [inline kube-router.go]: This change (splitting main into 2 functions) was so that I could call `main` in the test and not have it `os.Exit()` the test.

**Comment by murali-reddy** [comment]
@phemmer thanks. Indeed flags went outdated. Thanks for adding unit test as well, to make future proof.

**Comment by phemmer** [comment]
Fixed for failing build. For some reason `go test ./` generated different output than `go test github.com/cloudnativelabs/kube-router` (one of the items was annotated as `uints` vs `uintSlice`). Don't know why. Only reason I can figure is due to vendoring, but they should both use it.

**Comment by murali-reddy** [comment]
thanks @phemmer

**Pull request merged**